interface/chat/components/containers/chatbot-container.jsx

```diff
@@ -0,0 +1,76 @@
+import ChatOutput from "../chatbot/chat-output";
+import ChatInput from "../chatbot/chat-input";
+import SendChatInput from "../menu/chatbotButtons/send-chat-input";
+
+import ClearButton from "../menu/accountButtons/clear-button";
+
+import React, {useState} from "react";
+
+export default function Chatbot() {
+    // these have to be here or else it breaks :)
+    let formJson = null;
+    let currtext = null;
+
+    function handleSubmit(e) {
+            // prevents the browser from reloading the page
+        e.preventDefault();
+    
+            // reads the form data
+        const form = e.target;
+        const formData = new FormData(form);
+
+        const currtext = formData.get("inputArea");
+
+            // if input is empty, nothing happens
+        if (currtext === "") {
+            return;
+        }
+
+            // creates a new chat bubble w/ the input text inside of it, and puts it into the chat area
+        let newChatBubbleContainer = document.createElement("div");
+        newChatBubbleContainer.id = "chatBubbleContainer";
+        newChatBubbleContainer.className = "chat chat-end w-1/2 p-2 m-2 w-auto";
+        
+        let newChatBubble = document.createElement("div");
+        newChatBubble.id = "chatBubble";
+        newChatBubble.className = "chat-bubble bg-primary-light text-primary-darker flex-wrap";
+        newChatBubble.innerText = currtext;
+
+        newChatBubbleContainer.appendChild(newChatBubble);
+        chatArea.appendChild(newChatBubbleContainer);
+        
+            // clears the input area
+        inputArea.value = "";
+
+            // auto-scrolls to the bottom of the chat area
+        chatArea.scrollTop = chatArea.scrollHeight;
+        
+        
+        // You can pass formData as a fetch body directly:
+            //fetch('/some-api', { method: form.method, body: formData });
+    
+        // Or you can work with it as a plain object:
```

Comment:
is this needed? If so, add a line like this:

```
// TODO: XYZ needs to be done.
```
or remove it if its dead code.